### PR TITLE
Payment widget - Set status to pending after tokenization

### PR DESCRIPTION
**Pull request by bdvorachek**
# Critical Changes

# Changes
- After tokenization the Payment Status field on the Data Import record will be set to PENDING
# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

**lurch: attach W-040869

### CODE REVIEW

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001HuxMQAS" target="_blank">W-040869</a>

File: src/lwc/geConstants/geConstants.js [hunk 1]
```diff
@@ -9,9 +9,21 @@ const HTTP_CODES = Object.freeze({
     Request_Timeout: 408,
 });
 
+const PAYMENT_TRANSACTION_STATUS_ENUM = Object.freeze({
```
Comment (vmartinov-sforg) [inline src/lwc/geConstants/geConstants.js:12]: Before I review, please take a look at https://github.com/SalesforceFoundation/NPSP/pull/5395/files#diff-a2c9f4a7190565f47b4b3e2b3a6a9e31. We have decided to move the transaction status enum in apex in Daniel's PR.

File: src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js [hunk 2]
```diff
@@ -156,7 +158,10 @@ export default class geFormWidgetTokenizeCard extends LightningElement {
                     if (message.error) {
                         reject(this.handleTokenizationError(message));
                     } else if (message.token) {
-                        resolve({ [DATA_IMPORT_PAYMENT_AUTHORIZATION_TOKEN_FIELD.fieldApiName]: message.token });
+                        resolve({
+                            [DATA_IMPORT_PAYMENT_AUTHORIZATION_TOKEN_FIELD.fieldApiName]: message.token,
+                            [DATA_IMPORT_PAYMENT_STATUS_FIELD.fieldApiName]: PAYMENT_TRANSACTION_STATUS_ENUM.PENDING
```
Comment (psadie-sfdo) [inline src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js:163]: This is awesome once we align with however way we're retrieving the transaction status enums.

Comment (bdvorachek) [reply src/lwc/geFormWidgetTokenizeCard/geFormWidgetTokenizeCard.js]: We're aligned here now, pulling the pending value from the back-end.

**Pull request merged**